components/docs/DocsCards.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import { buckets } from '../data/buckets';
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+
+type IconType = (props: { className: string }) => JSX.Element;
```

Comment:
```suggestion
type IconType = (props: { className: string }) => React.Element;
```

Reply:
It it's giving error "type IconType = (props: { className: string }) => React.Element;". Should I use React.ReactElement?

Reply:
You can use `React.ReactNode`.

Reply:
Icons used are functional components and functional components `ReactElement<any, any> | null`. When I tried to use `React.ReactNode`, I got the following error:
```
'Icon' cannot be used as a JSX component.
 Its return type 'ReactNode' is not a valid JSX element.
 Type 'undefined' is not assignable to type 'Element | null'.ts(2786)
```
So I think we should use `React.ReactElement`?

Reply:
Hi @akshatnema, do you have any suggestions on this?